README.md

```diff
@@ -55,17 +57,16 @@ See [the latest GitHub release](https://github.com/pingidentity/pingcli/releases
 
 OR
 
-Use the following single-line powershell command to install Ping CLI into '%LOCALAPPDATA%\Programs' directly.
-```text
+Use the following single-line PowerShell 7.4 command to install Ping CLI into '%LOCALAPPDATA%\Programs' directly.
+```powershell
 $latestReleaseUrl = Invoke-WebRequest -Uri "https://github.com/pingidentity/pingcli/releases/latest" -MaximumRedirection 0 -ErrorAction Ignore -UseBasicParsing -SkipHttpErrorCheck; `
 $RELEASE_VERSION = [System.IO.Path]::GetFileName($latestReleaseUrl.Headers.Location); `
 $RELEASE_VERSION_NO_PREFIX = $RELEASE_VERSION -replace "^v", ""; `
-$uname = (uname -m); `
-$HARDWARE_PLATFORM = $uname -replace "aarch64", "arm64" -replace "x86_64", "amd64"; `
+$HARDWARE_PLATFORM = $env:PROCESSOR_ARCHITECTURE -replace "ARM", "arm64" -replace "ARM64", "arm64" -replace "x86", "amd64" -replace "AMD64", "amd64" -replace "EM64T", "amd64"; `
```

Comment:
`-replace "ARM", "arm64"` doesn't exist
`-replace "x86", "amd64"` should be `-replace "x86", "386"`